examples/amp-story/cta-layer-outlink.html

```diff
@@ -0,0 +1,226 @@
+<!doctype html>
+<html amp lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <script async custom-element="amp-story" src="https://cdn.ampproject.org/v0/amp-story-0.1.js"></script>
+  <title>My Story</title>
+  <meta name="description" content="Get started with amp-story">
+
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <link rel="canonical" href="grid-layer-templates.html">
+  <style amp-boilerplate>
```

Comment:
You should probably make this one line.  Try copying and pasting the whole doc into https://validator.ampproject.org and make sure it's valid

Reply:
Done. Now it only complains about the new <cta-layer> element since it's not supported by the validator yet. (Will be done in next PR).